lib/puppet/version.rb

```diff
@@ -72,6 +72,10 @@ def self.version=(version)
     @puppet_version = version
   end
 
+  def self.major_minor_version
+    self.version.sub(/\A(\d+\.\d+)\..*$/, '\1')
+  end
```

Comment:
I reviewed [PR-5720](/puppetlabs/puppet/pull/5720) and added a couple of comments there. I suggest we close this one and use that instead as it contains all needed tests.